OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -447,17 +447,8 @@ - (void)track:(NSString *)eventKey
         return;
     }
     
-    NSArray *decisions = [self decisionsFor:event userId:userId attributes:attributes];
-    
-    if ([decisions getValidArray] == nil) {
-        NSString *logMessage = [NSString stringWithFormat:OPTLYLoggerMessagesConversionFailure, eventKey];
-        [self handleErrorLogsForTrack:logMessage ofLevel:OptimizelyLogLevelInfo];
-        return;
-    }
-    
```

Comment:
All of https://github.com/optimizely/objective-c-sdk/blob/master/OptimizelySDKCore/OptimizelySDKCore/Optimizely.m#L912-L961 can be removed as well right?

Reply:
removed